server/config.go

```diff
@@ -207,9 +207,9 @@ func (c *Config) Validate(systemContext *types.SystemContext, onExecution bool)
 
 // Reload reloads the configuration with the config at the provided `fileName`
 // path. The method errors in case of any read or update failure.
-func (c *Config) Reload(systemContext *types.SystemContext, fileName string) error {
+func (c *Config) Reload(fileName string) error {
 	// Reload the config
-	newConfig, err := DefaultConfig(systemContext)
+	newConfig, err := DefaultConfig()
```

Comment:
Should we add an explicit `GetRegistries(systemContext)` here? I think it would be nice to keep that since a user gets immediate feedback when changing the configs and sending HUP to crio.

Reply:
Would that somehow incorporate with https://github.com/containers/image/pull/630?

Reply:
Good point, yes. We could either use that or invalidate and reload (for now).

Reply:
>I think it would be nice to keep that since a user gets immediate feedback when changing the configs and sending HUP to crio.

Without invalidation the `GetRegistries()` does not return updated data, so there is no feedback.

https://github.com/containers/image/pull/630 was created to benefit this location, but all of that is IMHO a new feature not really related to this initialization cleanup.